doc/documentation.rst

```diff
@@ -87,13 +87,6 @@ Documentation of scikit-learn |version|
             </div>
 
             <div class="row-fluid">
-                <div class="span4 box">
-                    <h2><a href="tutorial/machine_learning_map/index.html">Flow Chart</a></h2>
```

Comment:
Maybe remove Additional Resources instead? The flow chart seems useful from my side and most additional resources are outdated.

Reply:
@jnothman so you don't want to see the additional resources go?